controllers/util/version.go

```diff
@@ -0,0 +1,104 @@
+package util
```

Comment:
can you add a sample unit test on how you intend that one should use these util funcs? isn't the intention just to export a single func whether I can rotate the ceph key or not?